nbconvert/exporters/latex.py

```diff
@@ -43,6 +43,10 @@ def _template_skeleton_path_default(self):
     template_extension = Unicode(".tplx").tag(config=True)
 
     output_mimetype = 'text/latex'
+    
+    def default_filters(self):
+        yield from super(TemplateExporter,self).default_filters()
```

Comment:
For some reason even when running in my Python 3 environment, that doesn't work. with the current code. 

However, the Python 2 compatible way of doing of it does work. Any idea why?